### ISSUE TITLE
SA1009 invalid with new C# statements

### ISSUE THREAD

**Issue by diegoviscovig** [DotNetAnalyzers/StyleCopAnalyzers#2786]
```csharp
public async Task<(List<Email> sent, List<Email> fail)> SendEmailsAsync(List<Email> emails)
        {}
```

In this case stylecop suggested **"Closing parenthesis must be followed by a space".** and it isn't correct.
The error is after variable fail.

**Comment by sharwell** [comment]
Duplicate of #2537

**Comment by sharwell** [comment]
@diegoviscovig Thanks for taking the time to report this. You can get the fix by updating to the latest 1.1.0 beta (where C# 7 support was added).
